packages/dds/tree/src/simple-tree/treeApi.ts

```diff
@@ -164,6 +172,19 @@ export const treeNodeApi: TreeNodeApi = {
 			T
 		>;
 	},
+	shortId(node: TreeNode): number | undefined {
+		const flexNode = getFlexNode(node);
+		for (const field of flexNode.boxedIterator()) {
+			if (field.schema.kind === FieldKinds.identifier) {
+				const identifier = field.boxedAt(0);
+				assert(identifier !== undefined, "The identifier must exist");
+				return identifier.context.nodeKeys.localize(
+					identifier.value as StableNodeKey,
+				) as unknown as number;
+			}
+		}
+		return;
```

Comment:
unnecessary return. If you like the explicit return, then do `return undefined`.

Reply:
removed unnecessary return